### ISSUE TITLE
XMLTree and MATLAB 2018a error

### ISSUE THREAD

**Issue by DavidTingley** [buzsakilab/buzcode#206]
Looks like yet another naming change by Mathworks has broken LoadParameters.m

in 2018a, there is a convert.m file in some signal processing toolbox that takes priority over the convert.m used by XMLTree package.

**Comment by brendonw1** [comment]
Rename our convert.m to bz_convert.m?

**Comment by DavidTingley** [comment]
no, it is part of the XML toolbox.  We should add an explicit call to that function and path in Loadparameters.  I'll submit a fix hopefully later today

**Comment by DavidTingley** [comment]
When Manu gets around to it, we can test whether  8d94db1  fixes the issue.

**Comment by DavidTingley** [comment]
#207 will solve this issue
